apps/dashboard/app/helpers/projects_helper.rb

```diff
@@ -31,4 +31,8 @@ def button_category(status)
       status
     end
   end
+
+  def user_groups
+    CurrentUser.groups.map(&:name)
```

Comment:
CurrentUser already has this api.

https://github.com/OSC/ondemand/blob/c4d85369ef2afff04da1003aec66f7698b80cfc4/apps/dashboard/lib/current_user.rb#L39-L41